starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/ItemsObject.java

```diff
@@ -0,0 +1,49 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+/**
+ * Created by jay on 10/25/16. */
+
+public class ItemsObject {
```

Comment:
I suggest using name more specific to your app, like "ShoppingListItem" or "GroceryItem" rather than a very generic name. I know one of the examples in class used a very generic name, but that was just because it had very generic database entries that didn't represent any real-life thing. This app does represent a real-life thing though, so fortunately it is easier to give things more descriptive names.